### PR TITLE
Add instructions to handle `global` and `buffer` error

### PR DESCRIPTION
**Pull request by LieAlbertTriAdrian**

### CODE REVIEW

File: Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md [hunk 1]
```diff
@@ -108,6 +108,38 @@ import { Connection, PublicKey, clusterApiUrl } from '@solana/web3.js';
 import { Program, Provider, web3 } from '@project-serum/anchor';
 ```
```
Comment (AlmostEfficient) [inline Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md:110]: Please mention that this is only for Replit users.

Comment (LieAlbertTriAdrian) [reply Solana_And_Web3/en/Section_3/Lesson_2_Call_Deployed_Program.md:110]: Updated @AlmostEfficient

**Review by AlmostEfficient — APPROVED** [review]
Looks good, would be helpful to maybe have a link that explains these (if there is one)

**Comment by AlmostEfficient** [comment]
@all-contributors please add @LieAlbertTriAdrian for documentation

**Pull request merged**